apps/home/migrations/0030_auto_20220317_0939.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.12 on 2022-03-17 09:39
```

Comment:
Haven't renamed this as will change modal name once know if it's ok, also other story has lots of migrations so will need to redo this after rebase anyway